code/go/0chain.net/smartcontract/storagesc/config_settigns.go

```diff
@@ -585,6 +626,14 @@ func (conf *Config) set(key string, change string) error {
 		} else {
 			return fmt.Errorf("cannot convert key %s value %v to boolean: %v", key, change, err)
 		}
+	case smartcontract.Cost:
+		if value, err := strconv.Atoi(change); err == nil {
```

Comment:
avoid using nested ifs

Reply:
Yes, I did not like particularly but decided to copy the style of the other cases rather than rewrite the whole function.